IFC4x3/PropertyConstants/t/TRACKCIRCUITTUNINGZONESIGN_0ITvsmfOf6DOn8IhSNAIpL.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="TRACKCIRCUITTUNINGZONESIGN_0ITvsmfOf6DOn8IhSNAIpL" Name="TRACKCIRCUITTUNINGZONESIGN" UniqueId="12779db0-a58a-4635-8c48-4ab717292cd5" />
```

Comment:
Some of these types are quite complex to decipher due to their length.

Reply:
Done, I had underscore to make them more understandable!